src/thumbnailer_utils.cc

```diff
@@ -21,13 +21,13 @@ UtilsStatus AnimData2Pictures(WebPData* const webp_data,
   std::unique_ptr<WebPAnimDecoder, void (*)(WebPAnimDecoder*)> dec(
       WebPAnimDecoderNew(webp_data, NULL), WebPAnimDecoderDelete);
   if (dec == NULL) {
-    std::cerr << "Error parsing image." << std::endl;
+    std::cerr << "Error parsing image.\n";
```

Comment:
It is odd to mix the old C `"\n"` with the newer C++ `std::cout` syntax but I do not mind.